### DISCUSSION TITLE
v1.1.3 how to install

### DISCUSSION THREAD

**Discussion by zcb-code** [deepmodeling/deepmd-kit#1580] [Q&A] [answered]
I found a model trained with v.1.1.3, I want to check this model. But I can not install v1.1.3  with 
conda install deepmd-kit=1.1.3=*cpu lammps-dp=1.1.3=*cpu -c deepmodeling
How I can install  lammps-dp with deepmdkit for v1.1.3 ?
Thanks

**Comment by njzjz** [comment, marked as answer]
Instead, use the channel `-c https://conda.deepmodeling.org`

**Reply by zcb-code** [reply]
Thanks, it works.
